### PR TITLE
21473 Cleanup CharacterSet class

### PR DESCRIPTION
**Pull request by astares**
- remove unnecessary #empty method (as it has same impl. as in superclass)
- categorize as in superclass
- use returning ifTrue:ifFalse: in #includes:

https://pharo.fogbugz.com/f/cases/21473/Cleanup-CharacterSet-class

### CODE REVIEW

File: src/Collections-Support/CharacterSet.class.st [hunk 1]
```diff
@@ -142,12 +136,13 @@ CharacterSet >> hash [
...
-{ #category : #'collection ops' }
+{ #category : #testing }
 CharacterSet >> includes: aCharacter [
-	aCharacter asciiValue >= 256
+
+	^aCharacter asciiValue >= 256
 		ifTrue: ["Guard against wide characters"
-			^false].
-	^(map at: aCharacter asciiValue + 1) > 0
+					^false]
```
Comment (peteruhnak) [inline src/Collections-Support/CharacterSet.class.st]: This is no longer a guard, but a regular if/else; so the extra return is not needed.

**Review by peteruhnak — CHANGES_REQUESTED** [review]

File: src/Collections-Support/CharacterSet.class.st (hunk 1 above)
Comment (astares) [reply src/Collections-Support/CharacterSet.class.st]: Yes - it is not a guard using an early return anymore. Nonetheless the way it is written protects also from invalid access to the map - so functionality is given

Comment (akgrant43) [reply src/Collections-Support/CharacterSet.class.st]: Hi Torsten,

Can you explain how the extra return prevents invalid access to the map?  If it wasn't there, how could a wide character access the map?

Thanks,
Alistair

Comment (astares) [reply src/Collections-Support/CharacterSet.class.st]: I thought you questioned the transformation itself. 

s := CharacterSet newFrom: 'abc'.
s includes: 300 asCharacter 

Now I see that you meant the second ^ left on the true condition. 
This is just a leftover and not needed.

**Review by jecisc — APPROVED** [review]

File: src/Collections-Support/CharacterSet.class.st (hunk 1 above)
Comment (peteruhnak) [reply src/Collections-Support/CharacterSet.class.st]: Yes. Sorry, I should've been clearer.

**Review by peteruhnak — APPROVED** [review]

**Pull request merged**